crates/physical-plan/src/plan.rs

```diff
@@ -575,25 +711,14 @@ pub enum Sarg {
     Range(ColId, Bound<AlgebraicValue>, Bound<AlgebraicValue>),
```

Comment:
In 

```rust
pub enum Sarg {
     Range(ColId, Bound<AlgebraicValue>, Bound<AlgebraicValue>),
```
```suggestion
    Range(BinOp,ColId, Bound<AlgebraicValue>, Bound<AlgebraicValue>),
``` 

(for the plan printer)

Reply:
Can we defer this decision until we optimize range scans? I'm not sure which representation is best until we add rewrites that use it.

Reply:
I need the `BinOp` for printing the `plan`